exporter/kafkaexporter/kafka_exporter.go

```diff
@@ -122,6 +122,9 @@ func (e *kafkaLogsProducer) Close(context.Context) error {
 
 func newSaramaProducer(config Config) (sarama.SyncProducer, error) {
 	c := sarama.NewConfig()
+
+	c.ClientID = config.ClientID
```

Comment:
I don't see the default method changed to set it to `samara` by default

Reply:
yeah I was basically documenting the existing default in samara itself. Do you want me to set it as the default in the exporter as well? This felt like a smaller change, but happy to do that.

Reply:
Yes, please add the default to the [createDefaultConfig()](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/0c50ef5a58683a214a509ee15c75279a6334d9b9/exporter/kafkaexporter/factory.go#L90) method.